### DISCUSSION TITLE
What should we display on the Status Page, more latency or only uptime? 🤔

### DISCUSSION THREAD

**Discussion by thibaultleouay** [openstatusHQ/openstatus#608] [Ideas]
We use OpenStatus to monitor an endpoint from multiple regions (currently 6).
Our dashboard provides latency monitoring.
![image](https://github.com/openstatusHQ/openstatus/assets/13894054/6f9c8200-d1ee-4be8-b930-3ef197a3ca93)

Should these values be added to the status page?  
![CleanShot 2024-01-30 at 18 22 31](https://github.com/openstatusHQ/openstatus/assets/13894054/ad4cec12-86cf-4d03-8971-a41f23617fd4)

Currently on the Status Page, we display the total number of requests made during the day, along with the p95 value of these requests.

Do you want to display your latency or just know your uptime? Which one is more important to you?

Merci

**Comment by skushagra9** [comment]
yes,I think they must be added
